graphicsmagick/src/wand/magick.rs

```diff
@@ -1135,7 +1135,9 @@ impl<'a> MagickWand<'a> {
     /// MagickGetImageColorspace() gets the image colorspace.
     ///
     pub fn get_image_colorspace(&mut self) -> ColorspaceType {
-        unsafe { MagickGetImageColorspace(self.wand.as_ptr()) }.into()
+        unsafe { MagickGetImageColorspace(self.wand.as_ptr()) }
+            .try_into()
```

Comment:
We could use `into()` here if we add an `Unknown` variant to every enum in mod `types`.

This might also be good for backward compatiblity, @jmjoy what's your thought on this?

Reply:
On a second thought, we should definitely add an `Unknown` variant to archive forward compatibility.

Reply:
OK, this is good, because `Unknown` is equivalent to unreadable effect.